ztp/source-crs/ClusterVersion.yaml

```diff
@@ -5,8 +5,9 @@ metadata:
   annotations:
     ran.openshift.io/ztp-deploy-wave: "100"
 spec:
+  # The default channel is stable-<version>
   channel: $channel
-  desiredUpdate:
-    force: false
-    version: $version
-  upstream: https://api.openshift.com/api/upgrades_info/v1/graph # default upstream.
+  # The default upstream path is https://api.openshift.com/api/upgrades_info/v1/graph
+  upstream: $upstream
+  # desiredUpdate:
+  #   version: $version
```

Comment:
This needs to be commented out otherwise the generated policy will have "version: $version" if not overwriting the desiredUpdate version